apps/app/src/hooks/useCloseOnBreakpoint.ts

```diff
@@ -0,0 +1,28 @@
+import { Dispatch, SetStateAction, useCallback, useEffect, useState } from "react";
+
+type UseCloseOnBreakpointProps = {
+	initialState: boolean;
+	breakpoint: number;
+};
+
+export const useCloseOnBreakpoint = ({
```

Comment:
To chyba nie jest dobra nazwa, bo tu nic nie jest zamykane. Może useStateBelowBreakpoint albo coś takiego?

Reply:
Szczerze to jak zobaczysz historię to rzeczywiście brakowało mi pomysłu na tą nazwę 😂